feature/search/src/main/java/com/google/samples/apps/nowinandroid/feature/search/SearchScreen.kt

```diff
@@ -167,6 +171,16 @@ internal fun SearchScreen(
                         onInterestsClick = onInterestsClick,
                         searchQuery = searchQuery,
                     )
+                    if (recentSearchesUiState is RecentSearchQueriesUiState.Success) {
```

Comment:
Below is the current condition block:
```
is SearchResultUiState.Success -> {
    if (searchResultUiState.isEmpty()) {
        EmptySearchResultBody() // This object is not used if the below condition is true
        if (recentSearchesUiState is RecentSearchQueriesUiState.Success) {
            RecentSearchesBody()
        }
    } else {
        SearchResultBody()
    }
}
```

Should we change this to if/else so that we don't create `EmptySearchResultBody` needlessly?
```
if (recentSearchesUiState is RecentSearchQueriesUiState.Success) {
    RecentSearchesBody()
} else {
    EmptySearchResultBody()
}
```

Reply:
Thanks for the suggestion. Actually, as in the first screenshot of the CL comment, showing both the EmptySearchResultBody and RecentSearchesBody at the same time if the search result is empty is expected (The UI mock linked from design doc also suggest so).